src/maxdiffusion/multihost_dataloading.py

```diff
@@ -104,3 +104,30 @@ def get_next_batch_sharded(
   input_gdas = jtu.tree_map_with_path(partial(_form_global_array, global_mesh = global_mesh), local_data)
 
   return input_gdas
+
+def get_data_sharded(
```

Comment:
@suexu1025 is it possible to reuse the same code from [multihost_dataloading.py](https://github.com/google/maxdiffusion/blob/mlperf_4/src/maxdiffusion/multihost_dataloading.py)

Reply:
The existing can only take tf.data.Dataset.  I can just replace it with `jtu.tree_map_with_path(partial(_form_global_array, global_mesh = global_mesh), local_data)`